src/helm/proxy/clients/human_task_client.py

```diff
@@ -0,0 +1,99 @@
+from abc import ABC, abstractmethod
+
+import random
+from helm.common.hierarchical_logger import hlog
+from helm.common.cache import Cache, CacheConfig
+from helm.common.human_task_request import (
+    HumanQuestionTemplate,
+    HumanTaskRequest,
+    HumanTaskRequestResult,
+    HumanTaskTemplate,
+)
+from cattrs import unstructure
+import threading
+
+from typing import Dict, List
+import surge
+from surge import questions as surge_questions
+
+
+class HumanTaskClient(ABC):
```

Comment:
Can we also implement a LM-based version of this?  I actually think that would be very helpful for eval.

Reply:
OK, I am sufficiently convinced that we want this soon since OpenAI Evals is also doing it. But I will implement the LLM version in a different PR.

Reply:
Great, I think we'll want to try the LM version out before the Surge version just to pilot the questions.